gtsfm/utils/io.py

```diff
@@ -55,7 +55,7 @@ def load_image(img_path: str) -> Image:
         exif_data = parsed_data
 
     img_fname = Path(img_path).name
-    return Image(value_array=np.asarray(original_image), exif_data=exif_data, file_name=img_fname)
+    return Image(value_array=np.asarray(original_image.convert("RGB")), exif_data=exif_data, file_name=img_fname)
```

Comment:
why is this needed? Will this cause any regression elsewhere?

Reply:
If the source images are grayscale (as is that case for the AstroNet images) it will convert them to RGB. Otherwise, nothing changes.

Reply:
is the conversion expensive? or a no-op? maybe we should have a check to only do this if the image is grayscale?

Reply:
Good idea. Only converting if it's not already RGB is cleaner.

Reply:
Done